src/routes/MyPage.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from "react";
+import { InfoElement } from "../components/Info/InfoElement";
+import { getUser, getPosts } from "../apis/api";
+import { SmallPost } from "../components/Posts";
+
+const MyPage = () => {
+  const [email, setEmail] = useState("");
+  const [username, setUsername] = useState("");
+  const [college, setCollege] = useState("");
+  const [major, setMajor] = useState("");
+  const [postList, setPostList] = useState([]);
+  const [user, setUser] = useState(null);
+
+  useEffect(() => {
+    const getPostsAPI = async () => {
+      const posts = await getPosts();
+      setPostList(posts);
+      console.log(posts);
+    };
+    getPostsAPI();
+  }, []);
+
+
+  useEffect(() => {
+    const getInfoAPI = async () => {
+      const info = await getUser();
+      setEmail(info.user.email);
+      setUsername(info.user.username);
+      setCollege(info.college);
+      setMajor(info.major);
```

Comment:
```suggestion
      setFormData({
      email : info.user.email,
      username: info.user.username,
      college : info.college,
      major : info.major,
      });
```
위에서 이어진 내용으로, 저라면 이렇게 할 것 같습니다!